tests/channel_testing_utils.py

```diff
@@ -1,22 +1,23 @@
 import pathlib
 import os
 import socket
-import subprocess
 import sys
+import subprocess
+from typing import Tuple
 
 import pytest
 from xprocess import ProcessStarter
 
-from conda.testing.integration import _get_temp_prefix
+from conda.testing.integration import _get_temp_prefix, run_command
 
 
-def _mock_server(xprocess, name, port, auth="none", user=None, password=None, token=None):
+def _mock_http_server(xprocess, name, port, auth="none", user=None, password=None, token=None):
```

Comment:
Just tiny naming thing, since this is not related to the `mock` library I'd rather not use it for this fixture to reduce confusion when we use it in the future.